cdap-app-fabric/src/main/java/co/cask/cdap/security/hive/HiveTokenUtils.java

```diff
@@ -14,10 +14,14 @@
  * the License.
```

Comment:
Probably we should name it something else now since it also have the function which it depends on "getHiveClassloader" 
Or should we split in two classes ?